lib/katello/permission_creator.rb

```diff
@@ -231,6 +231,7 @@ def lifecycle_environment_permissions
       @plugin.permission :edit_lifecycle_environments,
```

Comment:
`edit_lifecycle_environments` permission is for editing the lifecycle environments themselves. I feel like the `edit_hosts` permission would be a better fit here. (Also, you can double check what permissions are needed when you change a host's content view in the web UI. We should be consistent.)

Reply:
@jeremylenz Updated. 

The permissions needed to edit a host's CV.
![permissions](https://github.com/user-attachments/assets/6d44cd8a-f0a7-4d3d-90b8-7793c640842d)